src/pip/_internal/vcs/subversion.py

```diff
@@ -33,6 +37,28 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def get_vcs_version(self):
+        # type: () -> Tuple[int, ...]
+        """Return the version of the currently installed Subversion client.
+
+        :return Tuple containing the parts of the version information.
+        :raises BadCommand: If ``svn`` is not installed.
+            ValueError: If the version returned from ``svn`` could not be
+            parsed.
+        """
+        version_pfx = 'svn, version '
```

Comment:
One more tiny style thing I meant to say. You can just write out `version_prefix` (I know you copied that from the Git version).